spring-boot-starters/langchain4j-open-ai-spring-boot-starter/pom.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>dev.langchain4j</groupId>
+        <artifactId>langchain4j-parent</artifactId>
+        <version>0.25.0-SNAPSHOT</version>
+        <relativePath>../../langchain4j-parent/pom.xml</relativePath>
+    </parent>
+
+    <artifactId>langchain4j-open-ai-spring-boot-starter</artifactId>
+    <name>LangChain4j Spring Boot starter for OpenAI</name>
+    <packaging>jar</packaging>
+
+    <dependencies>
+
+        <dependency>
+            <groupId>dev.langchain4j</groupId>
+            <artifactId>langchain4j-open-ai</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter</artifactId>
+        </dependency>
+
```

Comment:
You should add `spring-boot-autoconfigure-processor` here, have a look at https://docs.spring.io/spring-boot/docs/3.2.0/reference/htmlsingle/#features.developing-auto-configuration.custom-starter.autoconfigure-module

Reply:
Adding this dependency resulted in generating `spring-autoconfigure-metadata.properties` file with this content:
`dev.langchain4j.openai.spring.AutoConfig=`, not sure how does it help? This info is already present in `org.springframework.boot.autoconfigure.AutoConfiguration.imports` and `spring.factories` files...